idprovider/local/cache.go

```diff
@@ -81,17 +81,17 @@ func (c *contactsCache) ReadByOffChainAddr(offChainAddr pwire.Address) (_ perun.
 	return c.readByAlias(alias)
 }
 
-// Write adds the peer to contacts cache. Returns an error if the alias is already used by same or different peer or,
-// if the off-chain address string of the peer cannot be parsed using the wallet backend of this contacts provider.
-func (c *contactsCache) Write(alias string, p perun.Peer) error {
+// Write adds the peer to idProvider cache. Returns an error if the alias is already used by same or different peer or,
+// if the off-chain address string of the peer cannot be parsed using the wallet backend of this idProvider.
```

Comment:
```suggestion
// if the off-chain address string of the peer cannot be parsed using the wallet backend of this ID Provider.
```

Using ID provider as two words is more readable in the context of comments and documentation.